packages/cli/lib/services/dryrun.service.ts

```diff
@@ -372,6 +379,25 @@ export class DryRunService {
                         break;
                     }
                 }
+
+                if (options.saveResponses) {
+                    const directoryName = `${process.env['NANGO_MOCKS_RESPONSE_DIRECTORY']}${providerConfigKey}`;
+                    if (!fs.existsSync(directoryName)) {
+                        fs.mkdirSync(`${directoryName}/mocks`, { recursive: true });
+                    }
```

Comment:
Unecessary if you are creating another folder recursively just after

Reply:
Updated